Loop/Managers/Alerts/DeviceAlertManager.swift

```diff
@@ -146,29 +152,7 @@ extension DeviceAlertManager {
     
 }
 
-extension FileManager {
```

Comment:
This code moved, unchanged, below all of the `extension DeviceAlertManager` declarations